### DISCUSSION TITLE
Klik tombol Mark as Done

### DISCUSSION THREAD

**Discussion by purwto3369** [teamalgoritma/community#186] [Q&A] [answered]
Pada pengumpulan dan penilaian capstone terdapat : Klik tombol **Mark as Done**, dimana saya bisa temukan setelah saya selesai Upload file capston-ui-purwto di github.com?

**Comment by yosiaazarya** [comment, marked as answer]
Mark as Done dapat bapak temukan di google classroom Bapak. Setelah bapak berhasil upload, kirimkan link repository (contoh : https://github.com/yosiaazarya/capstone_da_telebot) case capstone bapak ke private message di assignment capstone data analytics dan mark as done di assignment terkait.
